### PR TITLE
MC-19865: Updated their api docs to include the change made to the ne…

### PR DESCRIPTION
**Pull request by fyrnaga**
…twork policies

#### Changes made

- Updated documentation to include the last change to the network policies

#### Related PRs
- PR # https://github.com/cloudops/cloudmc-api-docs/pull/596

### CODE REVIEW

**Review by patrickspensieri — APPROVED** [review]
Minor, but we can use a capital T on "the list of..."

File: source/includes/edgecompute/_network_policy_rules.md [hunk 1]
```diff
@@ -58,14 +58,14 @@ Retrieve a list of all network policy rules in a given [environment](#administra
...
 | `workloadId`<br/>_UUID_      | The UUID of the workload to which the network policy rule is applied. Corresponds to the first workload ID in the network policy's list of instance selectors. |
 | `networkPolicyId`<br/>_UUID_ | The UUID of the network policy to which the network policy rule belongs.                                                                                       |
 | `description`<br/>_string_   | A summary of what this rule does or a name of this rule. It is highly recommended to give a unique description to easily identify a rule.                      |
 | `type`<br/>_string_          | The type of network policy rule, either `INBOUND` or `OUTBOUND`.                                                                                               |
-| `source`<br/>_string_        | A subnet that will define all the IPs allowed or denied by this rule.                                                                                          |
+| `source`<br/>_Array[string]_ | the list of subnets that will define all the IPs allowed or denied by this rule.                                                                               |
```
Comment (HassanHaidar) [inline source/includes/edgecompute/_network_policy_rules.md]:
```suggestion
| `source`<br/>_Array[string]_ | The list of subnets that will define all the IPs allowed or denied by this rule.                                                                               |
```

File: source/includes/edgecompute/_network_policy_rules.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ Retrieve a network policy rule in a given [environment](#administration-environm
...
```
Comment (HassanHaidar) [inline source/includes/edgecompute/_network_policy_rules.md]:
```suggestion
| `source`<br/>_Array[string]_ | The list of subnets that will define all the IPs allowed or denied by this rule.                                                                               |
```

File: source/includes/edgecompute/_network_policy_rules.md [hunk 3]
```diff
@@ -152,7 +152,7 @@ Create a new network policy rule.
 | `protocol`<br/>_string_    | Supported protocols are: `TCP`, `UDP`, `TCP_UDP`, `ESP` and `AH`.                                                                         |
 | `type`<br/>_string_        | The type of network policy rule. Supported types are: `INBOUND` (Ingress) and `OUTBOUND` (Egress).                                        |
 | `action`<br/>_string_      | The network policy rule action: `ALLOW` (allow traffic) or `BLOCK` (block traffic).                                                       |
-| `source`<br/>_string_      | A subnet that will define all the IPs allowed or denied by this rule.                                                                     |
+| `source`<br/>_Array[string]_ | the list of subnets that will define all the IPs allowed or denied by this rule.                                                                               |
```
Comment (HassanHaidar) [inline source/includes/edgecompute/_network_policy_rules.md]:
```suggestion
| `source`<br/>_Array[string]_ | The list of subnets that will define all the IPs allowed or denied by this rule.                                                                               |
```

File: source/includes/edgecompute/_network_policy_rules.md [hunk 4]
```diff
@@ -203,7 +203,7 @@ Edit a network policy rule.
 | `description`<br/>_string_ | A summary of what this rule does or a name of this rule. It is highly recommended to give a unique description to easily identify a rule.                                    |
 | `workloadId`<br/>_UUID_    | The UUID of the workload to which the network policy rule is applied. Corresponds to the first workload ID in the network policy's list of instance selectors.               |
 | `type`<br/>_string_        | The type of network policy rule. Supported types are: `INBOUND` (Ingress) and `OUTBOUND` (Egress).                                                                           |
-| `source`<br/>_string_      | A CIDR subnet that will define all the IPs allowed or denied by this rule.                                                                                                   |
+| `source`<br/>_Array[string]_ | the list of subnets that will define all the IPs allowed or denied by this rule.                                                                               |
```
Comment (HassanHaidar) [inline source/includes/edgecompute/_network_policy_rules.md]:
```suggestion
| `source`<br/>_Array[string]_ | The list of subnets that will define all the IPs allowed or denied by this rule.                                                                               |
```

**Review by HassanHaidar — APPROVED** [review]
Just some capitalization needed

File: source/includes/edgecompute/_network_policy_rules.md (hunk 1 above)
Comment (fyrnaga) [reply source/includes/edgecompute/_network_policy_rules.md]: 7663a9e984321793e498c036460e0832d16ad8e3

File: source/includes/edgecompute/_network_policy_rules.md (hunk 2 above)
Comment (fyrnaga) [reply source/includes/edgecompute/_network_policy_rules.md]: 7663a9e984321793e498c036460e0832d16ad8e3

File: source/includes/edgecompute/_network_policy_rules.md (hunk 3 above)
Comment (fyrnaga) [reply source/includes/edgecompute/_network_policy_rules.md]: 7663a9e984321793e498c036460e0832d16ad8e3

File: source/includes/edgecompute/_network_policy_rules.md (hunk 4 above)
Comment (fyrnaga) [reply source/includes/edgecompute/_network_policy_rules.md]: 7663a9e984321793e498c036460e0832d16ad8e3

**Pull request merged**